### PR TITLE
Add api versionning

### PR DESCRIPTION
**Pull request by niio972**
- add api resource service for api mininal description

### CODE REVIEW

File: phis2-ws/src/main/java/opensilex/service/resource/dto/APIDescriptionDTO.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    /**
+     * Name of the API
+     * @example OpenSILEX API
+     */
+    public String name;
+    /**
+     * The build identifier
+     * @example 
+     */
+    public String build;
```
Comment (annetireau) [inline phis2-ws/src/main/java/opensilex/service/resource/dto/APIDescriptionDTO.java]: what is the build in this context ?

File: phis2-ws/pom.xml [hunk 2]
```diff
@@ -4,11 +4,14 @@
...
-    <version>v0.1</version>
+    <version>v3.3.0</version>
     <packaging>war</packaging>
     <name>opensilex</name>
-
+    
     <properties>
+        <apiName>OpenSILEX API</apiName>
+        <name>${project.name}</name>
+        <version>${project.version}</version>
```
Comment (annetireau) [inline phis2-ws/pom.xml]: Could you get data from profile ?

Comment (annetireau) [reply phis2-ws/pom.xml]: or vice versa ?

Comment (niio972) [reply phis2-ws/pom.xml]: Maybe but it will be more interesting to get it from the pom file.

Comment (annetireau) [reply phis2-ws/pom.xml]: You also can get apiName from ws.target

Comment (annetireau) [reply phis2-ws/pom.xml]: And also add version

File: phis2-ws/src/main/java/opensilex/service/resource/ApiResourceService.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+@Path("/api")
+public class ApiResourceService extends ResourceService {
+
+    /**
+     * Describe API state
+     * @return list of API descritpions
+     * @example
+     * {
+     *   "majorVersion": 3,
+     *   "buildVersion": "v3.3.0",
```
Comment (annetireau) [inline phis2-ws/src/main/java/opensilex/service/resource/ApiResourceService.java]: remove "v"

File: phis2-ws/pom.xml (hunk 2 above)
Comment (niio972) [reply phis2-ws/pom.xml]: Retreive api name from ws.target. Add ApiVersion variables to profiles files.

File: phis2-ws/src/main/java/opensilex/service/resource/ApiResourceService.java (hunk 3 above)
Comment (niio972) [reply phis2-ws/src/main/java/opensilex/service/resource/ApiResourceService.java]: Removed

**Pull request merged**